CMakeLists.txt

```diff
@@ -10,8 +10,15 @@ set(CMAKE_FORTRAN_STANDARD 08)
 set(CMAKE_FORTRAN_STANDARD_REQUIRED ON)
 set(CMAKE_FORTRAN_EXTENSIONS OFF)
 
-find_package(NetCDF REQUIRED COMPONENTS CXX)
-find_package(oops REQUIRED)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+
+find_package(NetCDF REQUIRED COMPONENTS CXX Fortran)
+#find_package(oops REQUIRED)
```

Comment:
I think I want to make this a bit more flexible. Do you want me to push to your branch, create my own branch, or another option?

Reply:
Whatever suits you.

Reply:
I just realized this is on a fork, so I'll open a new PR to fix the CMake stuff only